client.go

```diff
@@ -39,7 +39,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/hashicorp/go-cleanhttp"
+	cleanhttp "github.com/hashicorp/go-cleanhttp"
```

Comment:
This is an unnecessary change.

Reply:
It is unrelated but is intentional as it fixes the auto-removal of this import by goimports because in this case, the import path does not match the package name. Ref: https://github.com/golang/go/issues/28428

This slightly improves readability with no functional repercussions. I will remove this if you insist. Thank you.

cc @jefferai @evanphx

Reply:
So that's strange because imports prefixed by `go-` are auto-added just fine for me. That's why I suggested this wasn't necessary. I actually just tried taking a file and removing the `cleanhttp` and running `gomiports` on the file and it leaves it alone. Is it possible you're using a very old version of goimports or that you're not using go modules? IIRC this started working properly right around when go modules dropped on the scene.

Reply:
Sorry, my bad. You're right. I was using go modules but I had an older version of goimports.